### PR TITLE
hub: refresh quickstart

### PR DESCRIPTION
**Pull request by craig-osterhout**
## Description

Refreshed Hub Quickstart.
 - Added a brief intro.
 - Added the "how to find an image" use-case.
 - Updated pull/push steps to use the same image to show how easy it is to extend images.
 - Moved install DD and create an account as prereqs to keeps the steps Hub-focused.
 - Updated next steps to have a more natural flow related to the previous step.
 
 https://deploy-preview-21374--docsdocker.netlify.app/docker-hub/quickstart/
 
## Related issues or tickets

ENGDOCS-2227

## Reviews

- [ ] Editorial review
- [ ] Product review

### CODE REVIEW

File: content/manuals/docker-hub/quickstart.md [hunk 1]
```diff
@@ -1,148 +1,221 @@
...
-You need to download Docker Desktop to build, push, and pull container images.
+   After the results are displayed, you can further filter the results using the
+   filters on the left side of the page.
 
-1. Download and install [Docker Desktop](/manuals/desktop/_index.md).
+3. In the filters, select **Docker Official Image**.
 
-2. Sign in to Docker Desktop using the Docker ID you created in step one.
+   Filtering by Trusted Content ensures that you see only high-quality, secure
+   images, curated by Docker and verified publishing partners.
```
Comment (sarahsanders-docker) [inline content/manuals/docker-hub/quickstart.md:43]:
```suggestion
   images curated by Docker and verified publishing partners.
```

File: content/manuals/docker-hub/quickstart.md [hunk 2]
```diff
@@ -1,148 +1,221 @@
...
 
-### Step 4: Pull and run a container image from Docker Hub
+4. In the results, select the Nginx image.
```
Comment (sarahsanders-docker) [inline content/manuals/docker-hub/quickstart.md:45]: more of a question/nit, should **Ngix** or **Ngix image** be bold if they are clicking on it?

File: content/manuals/docker-hub/quickstart.md [hunk 3]
```diff
@@ -1,148 +1,221 @@
...
 
-1. In your terminal, run `docker pull hello-world` to pull the image from Docker Hub. You should see output similar to:
+   Selecting the image opens the image's page where you can learn more about how
+   to use the image. In the page, you'll also find the `docker pull` command to
```
Comment (sarahsanders-docker) [inline content/manuals/docker-hub/quickstart.md:48]:
```suggestion
   to use the image. On the page, you'll also find the `docker pull` command to
```

File: content/manuals/docker-hub/quickstart.md [hunk 4]
```diff
@@ -1,148 +1,221 @@
...
+   ```console
+   $ docker run -p 8080:80 --rm <YOUR-USERNAME>/nginx-custom
    ```
 
-2. Run `docker build -t <your_username>/my-private-repo .` to build your Docker
-   image.
+4. Visit [https://localhost:8080](https://localhost:8080) to view the page. You
+   should see `Hello world from Docker!`.
+
+5. In the terminal, press ctrl+c to stop the container.
```
Comment (sarahsanders-docker) [inline content/manuals/docker-hub/quickstart.md:164]:
```suggestion
5. In the terminal, press CTRL+C to stop the container.
```

**Review by sarahsanders-docker — CHANGES_REQUESTED** [review]
left a few comments w/ suggestions & a question about bolding

File: content/manuals/docker-hub/quickstart.md (hunk 2 above)
Comment (craig-osterhout) [reply content/manuals/docker-hub/quickstart.md:45]: Yea, I was trying to appease vale. I'll bold it and make it literal UI text. The literal UI text is `nginx`, but vale nags about the capitalization.

**Review by sarahsanders-docker — APPROVED** [review]
LGTM!

**Pull request merged**